packages/react-core/src/core/providers/thirdweb-wallet-provider.tsx

```diff
@@ -54,6 +54,9 @@ type ConnectFnArgs<I extends WalletInstance> =
         connectParams: NonNullable<WalletConnectParams<I>>,
       ];
 
+// maps wallet instance to it's wallet config
+const walletInstanceToConfig: Map<WalletInstance, ConfiguredWallet> = new Map();
```

Comment:
Is this just to optimize not having to search for the config?

Reply:
not because of that - but we can't search it - since the id in config.meta and wallet instance can be different